webpack.config.js

```diff
@@ -7,6 +7,7 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 var ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
 var externalModules = [
+	'async_hooks',
```

Comment:
This is to make webpack on node 6 happy with the bluebird update, bluebird only loads `async_hooks` when it's supported so it's fine at runtime but was causing webpack issues when trying to resolve the module for node 6 (as it doesn't exist there)